### PR TITLE
Add tests and Docs for Constants and Ambient files

### LINKED ISSUE #215: Ambient class need tests and documentation

### LINKED ISSUE #216: constants.py need tests and docs

### PR DESCRIPTION
**Pull request by joaofoes**
Fixes #216 
Fixes #215

### CODE REVIEW

**Comment by joaofoes** [comment]
First of all i want to apologize, i was making both tests all at once and when i realized i've pushed them in the same PR, sorry for that.

Second, about the test for ambient.py. 
I'm in doubt on how to create a test for the case of Raise ValueError.

File: docs/World/Ambient.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+ambient = Ambient(temperature = 300, pressure = 101325, humidity = -30)
+
+print(ambient.air_density)
+>>> raise ValueError(
+                f"Absolute temperature and absolute pressure should be greater than zero. \
+                Humidity should be between 0 and 100. \
+                Found temp = 300, pressure = 101325 and humidity = -30"
+            )
+```
```
Comment (rafaellehmkuhl) [inline docs/World/Ambient.md:37]: Great documentation!

File: docs/World/constants.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Constants
+
+This file will return you the values of Air Molar Mass, Gravitacional Acceleration and Air Gas Constant.
```
Comment (rafaellehmkuhl) [inline docs/World/constants.md:3]: I think a better explanation would be to say that there are several constants available there, and list the names of the constants and their values with a table, like so:

| Name | Value |
|------|-------|
| c1   | v1    |
| c2   | v2    |

You can generate such tables [with this website](https://www.tablesgenerator.com/markdown_tables).

File: tests/World/test_constants.py [hunk 3]
```diff
@@ -1,5 +1,14 @@
...
-    pass
+    assert constants.air_molar_mass == 0.02896
+
+
+def test_gravitacional_acceleration():
+    assert constants.gravitational_acceleration == 9.80665
+
+
+def test_air_gas_constant():
+    npt.assert_almost_equal(constants.air_gas_constant, 287.101, decimal=3)
```
Comment (rafaellehmkuhl) [inline tests/World/test_constants.py:14]: Maybe for easing on dev time, we can have the tests of all fixed constants on one function, and the tests for the calculated ones on another. No need for a separate function on each constant.

File: tests/World/test_Ambient.py [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+@pytest.fixture
+def base_ambient():
+    base_ambient = Ambient(temperature=288.15, pressure=101325, humidity=30)
+    return base_ambient
+
+
+@pytest.fixture
+def base_ambient_ERROR():
+    base_ambient_ERROR = Ambient(temperature=288.15, pressure=101325, humidity=-30)
+    return base_ambient_ERROR
```
Comment (rafaellehmkuhl) [inline tests/World/test_Ambient.py:15]: You can test for errors on a test_instantiation_with_error(), not using the fixtures, like so:

```
ambient = Ambient(temperature=0)
with raises(ValueError) as e_info:
    air_density = ambient.air_density

ambient = Ambient(humidity=104)
with raises(ValueError) as e_info:
    air_density = ambient.air_density
```

When you are inside a "with raises(Error)" context pytest will verify if the code inside it (air_density = ambient.air_density) raises the given error. If it does, the test passes, if not, it won't.

**Comment by rafaellehmkuhl** [comment]
And no need to apologise.
Anyway we will make a rebase in the end, so if the commits are well separated we are good.

File: tests/World/test_Ambient.py [hunk 5]
```diff
@@ -24,3 +18,17 @@ def test_instantiation(base_ambient):
...
+        air_density = ambient.air_density
+
+    ambient = Ambient(humidity=104)
+    with pytest.raises(ValueError) as e_info:
+        air_density = ambient.air_density
+
+    ambient = Ambient(pressure=-3)
+    with pytest.raises(ValueError) as e_info:
+        air_density = ambient.air_density
+
```
Comment (rafaellehmkuhl) [inline tests/World/test_Ambient.py:34]: Nice!

**Comment by rafaellehmkuhl** [comment]
Joao, it's all great now.
Just rebase, squash the commits and push force so we can merge them.

**Pull request merged**